include/swift/AST/Pattern.h

```diff
@@ -50,6 +50,21 @@ enum : unsigned { NumPatternKindBits =
 /// Diagnostic printing of PatternKinds.
 llvm::raw_ostream &operator<<(llvm::raw_ostream &OS, PatternKind kind);
```

Comment:
Just noticed that this method can also be removed (and its implementation in Pattern.cpp)